metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/feed/security/JcrFeedAllowedActions.java

```diff
@@ -122,12 +131,17 @@ protected void updateEntityAccess(Principal principal, Set<? extends Action> act
         Set<String> detailPrivs = new HashSet<>();
         Set<String> dataPrivs = new HashSet<>();
         Set<String> summaryPrivs = new HashSet<>();
+        
+        // Enable/disable feed ops access
+        if (actions.stream().filter(action -> action.implies(FeedAccessControl.ACCESS_OPS)).findFirst().isPresent()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "filter().findFirst().isPresent()" chain with "anyMatch()" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4034)